src/puppetlabs/puppetdb/query/event_counts.clj

```diff
@@ -116,10 +116,11 @@
   "Helper function to transform the event count subject data from the raw format that we get back from the
   database into the more structured format that the API specifies."
   [summarize_by]
-  (fn [rows]
-    (mapv
-     (partial munge-subject summarize_by)
-     rows)))
+  (fn [_ _ _ _] 
```

Comment:
Seems like this should be constantly since that's what you're using everywhere else